admin/yum.py

```diff
@@ -123,7 +123,8 @@ def perform_create_repository(dispatcher, intent):
         metadata = check_output([
             b'dpkg-scanpackages',
             b'--multiversion',
-            intent.repository_path.path])
```

Comment:
:question: Put a comment before each option to explain what it does - I've found this useful elsewhere in our source.